mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogFragment.java

```diff
@@ -67,6 +67,9 @@
     @BindView(R.id.tv_choose_file)
     TextView tv_choose_file;
 
+    @BindView(R.id.btn_browse_document)
+    Button bt_browse;
```

Comment:
Please follow the camel case naming convention

Reply:
@therajanmaurya snake case convention is followed across the entire document. For example : tv_choose_file,et_document_description

Reply:
@therajanmaurya done